Screensaver-world/CornerStore
# DNS Profiles Integration

## Summary
- adds hooks for loading profile data from Ceramic
- updates to node v14 on Netlify
- include profile data on profile page
- try to load avatar images where used throughout the site (I may have missed 1 or 2 places)
- update the edit profile page to link to DNS.xyz

Design (left=old, right=new): https://www.figma.com/file/VbyrQs6YcRlqZfAcQZlxX4/DNS-Edit-Profile?node-id=0%3A1

## Testing
1) Login with your ETH wallet
2) Go to your profile page (see the Call To Action to create your profile)
3) Go to `/profile/0x5984c906dbdf0a5aed47a202138f3163e2d3bd4e` to see my test profile
4) Go to `/profile/edit` to see the description of DNS.xyz (let me know if you have any questions)
